packages/core/botpress/tests/bot.js

```diff
@@ -0,0 +1,127 @@
+import { expect } from 'chai'
+import path from 'path'
+import { existsSync } from 'fs'
+import { exec, execSync } from 'child_process'
+import moment from 'moment'
+
+const bin = path.resolve(__dirname, '../bin/botpress')
+const botDir = path.resolve(__dirname, '../tmp-bot')
+const botConfig = path.resolve(__dirname, '../tmp-bot/config')
+const botpressDir = path.resolve(__dirname)
+const rootDir = path.resolve(__dirname, '../../../../')
+const compileLinkBash = path.resolve(rootDir, './compile-link.sh')
+const configVar = require('./config/env.json')
+
+const botpressModules = [
+  '@botpress/analytics',
+  '@botpress/audience',
+  '@botpress/broadcast',
+  '@botpress/hitl',
+  '@botpress/nlu',
+  '@botpress/qna',
+  '@botpress/scheduler',
+  '@botpress/terminal',
+  '@botpress/channel-messenger',
+  '@botpress/channel-slack',
+  '@botpress/channel-telegram',
+  '@botpress/channel-twilio',
+  '@botpress/channel-microsoft',
+  '@botpress/builtins',
+  '@botpress/util-roles',
+  '@botpress/skill-choice',
+  'botpress'
+]
+
+const LAUNCHED = /Bot launched/gi
+
+describe('Create new bot', function() {
+  this.timeout(0)
+
+  let hasError = false
+
+  const ifErrorStopDescribe = error => {
+    if (hasError || error) {
```

Comment:
do we actually need to check `hasError` here? if `error` is true-ish `hasError` check is not needed, and if `hasError` is `true` but there's no `error` we should fail from the corresponding test, not from here

Am I missing something?